treebeard/mp_tree.py

```diff
@@ -374,7 +375,7 @@ def process(self):
                       ' and UPDATE your database'))
         else:
             # adding the new child as the last one
-            newobj.path = self.node.get_last_child()._inc_path()
```

Comment:
it looks like what's happening here is the `self.numchild` attribute has been updated but the database doesn't have the child there

Reply:
thanks to me (;

Reply:
?